### PR TITLE
Mobile nav fixes, removed search, and updated google tag manager

### LINKED ISSUE #352: Take inventory of mobile fixes needed
Go through all of the pages on mobile device (multiple phone types & versions if possible) to see what needs addressing for small screens, and create GH Issues to fix.

Known: 

- [x] 1. Tabs are working on mobile, but are expanding slightly off the page
- [x] 2. top nav dropdowns are overlapping nav items, instead of pushing them down
- [x] 3. footer/jumbotron is not full length for iPhone 5
- [x] 4. blog card is longer than size of mobile screen
- [x] 5. docs banner's height needs to be increased (index and blog page)

### LINKED ISSUE #476: Dropdowns in top-nav do not cover other nav items
When opening the dropdowns in the top nav, they are overlapping the items below in the nav instead of pushing the items down. All nav items should remain visible when dropdowns are expanded.

### PR DESCRIPTION
**Pull request by typaulhus**
**What does this PR do?**
1. On mobile screens, open dropdown menus in the top nav are overlapping other items in the navigation.
- Closes #476 
- Closes #352 
2. Fixes errors due to multiple Google tag manager & analytics instances
3. Removes WIP search functionality

**How should I test?**
1. Open web inspector, and confirm there are no errors pertaining to google tag manager (should be no errors for anything)
2. Confirm that the magnifying glass icon is removed from the top nav, and manually navigating to /search does not go to a search page
3. Confirm that when viewing at mobile screen widths, opening the dropdown menu does not overlap other navigation items

**Screenshots**
Before (nav items are hidden under blue drop down menu):
screenshot_2018-11-06 secretless broker secretless 1

After (happy happy joy joy):
screenshot_2018-11-06 secretless broker secretless

Search removed
Search icon was previously in top nav. This was only appearing in Staging
screenshot_2018-11-09 community secretless

### CODE REVIEW

**Comment by sigalsax** [comment]
Everything looks great but except on my end I still see the search magnifying glass in the navbar menu. Do you see this on your end?

File: docs/_sass/_variables.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 $sless-dark-blue: #08141C;
 $sless-blue: #4D8FCC;
 $sless-light-blue: #EDF2F6;
-$border-radius: .4rem;
+$border-radius: 3px;
```
Comment (izgeri) [inline docs/_sass/_variables.scss]: aren't we trying to avoid px?

File: docs/_sass/_navbar.scss [hunk 2]
```diff
@@ -89,13 +91,19 @@
 .navbar-nav {
 	.dropdown-menu {
 		width: 100%;
+		.top-drop {
+			border-bottom: 1px solid #ffffff1a;
```
Comment (izgeri) [inline docs/_sass/_navbar.scss]: why px instead of rem? it should be consistent with rest of file

File: docs/_sass/_variables.scss (hunk 1 above)
Comment (typaulhus) [reply docs/_sass/_variables.scss]: yea, will change these to rem

**Comment by typaulhus** [comment]
@izgeri @sigalsax I addressed the comments (thanks for the feedback!) and also removed the search, so there's no need to revert anything.

**Comment by sigalsax** [comment]
these changes were merged into staging. Did we want to also merge into master?

**Pull request merged**